superset/security.py

```diff
@@ -77,177 +78,303 @@
 ])
 
 
-def merge_perm(sm, permission_name, view_menu_name):
```

Comment:
It probably merits exampling in the PR description what's going here, i.e., the `SupersetSecurityManager` class in a combination of the previously defined methods in `security.py` in combination with accessor methods previously defined in the `BaseSupersetView` class which aren't view specific (_hypothesizing_).

Reply:
Will do